### PR TITLE
develop tier feature update

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 1]
```diff
@@ -16,17 +16,30 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
 
 1. 如果你还未拥有 TiDB Cloud 帐号，请先在此[注册](https://tidbcloud.com/free-trial)。
 2. 使用你的 TiDB Cloud 帐号[登录](https://tidbcloud.com/)。
-3. 在[方案](https://tidbcloud.com/console/plans)内选择一年内免费的 Developer Tier 方案，或直接点击[创建 Dev Tier 集群](https://tidbcloud.com/console/create-cluster?tier=dev)，进入 **Create a Cluster (Dev Tier)** 页面。
-4. 请在 **Create a Cluster (Dev Tier)** 页面填写集群名称/密码/云服务商（暂时仅可选择 AWS）/ 可用区（建议就近选择）后，点击 **Create** 按钮创建集群。
-5. 稍作等待，在 5~15 分钟后，将创建完毕，可在 [Active Clusters](https://tidbcloud.com/console/clusters) 查看创建进度。
+3. 在[方案](https://tidbcloud.com/console/plans)内选择一年内免费的 **Developer Tier** 方案。
+4. 请在 **Create a Cluster** 页面填写集群名称/密码/云服务商（暂时仅可选择 AWS）/ 可用区（建议就近选择）后，点击 **Create** 按钮创建 Developer Tier 免费集群。
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:20]:
```suggestion
4. 在 **Create Cluster** 页面设置集群名称、云服务商（Developer Tier 默认为 AWS）、可用区（建议就近选择）后，点击 **Create** 按钮创建 Developer Tier 免费集群。
```

Comment (qiancai) [reply develop/dev-guide-build-cluster-in-cloud.md:20]: 密码已经从 **Create Cluster** 页面 去掉了

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 2]
```diff
@@ -16,17 +16,30 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
...
+
+    这将开始集群的创建过程，并且会弹出 **Security Quick Start** 对话框。
+
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:23]:
```suggestion
```

Comment (qiancai) [reply develop/dev-guide-build-cluster-in-cloud.md:23]: 中文看起来不要这句话更简洁。

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 3]
```diff
@@ -16,17 +16,30 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
...
+5. 在 **Security Quick Start** 对话框，设置密码，并允许 IP 地址来连接你的集群，完成时点击 **Apply**。
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:24]:
```suggestion
5. 在 **Security Quick Start** 对话框中，设置密码，并添加允许连接你的集群的 IP 地址，完成后点击 **Apply**。
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 4]
```diff
@@ -16,17 +16,30 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
...
-![SQL string](/media/develop/IMG_20220331-232800929.png)
+    ![connect](/media/develop/connect.jpg)
+
+8. 复制弹出框 **Step 2: Connect with a SQL client** 中的连接字符串，供后续步骤使用。
+
+    ![SQL string](/media/develop/sql-string.jpg)
+
+    > **Note:**
+    >
+    > - 需要特别说明的是，在你使用 [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier) 集群时，你需要给你设置的用户名加上前缀（如上图中的 `4JC1i9KroBMFRwW`），若使用命令行连接，还需使用单引号包裹用户名. 你可以在 [TiDB Cloud - 用户名前缀](/tidb-cloud/select-cluster-tier.md#user-name-prefix) 中获得更多信息。
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:42]:
```suggestion
    > - 需要特别说明的是，在你使用 [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier) 集群时，你需要给你设置的用户名加上前缀（如上图中的 `4JC1i9KroBMFRwW`），若使用命令行连接，还需使用单引号包裹用户名。你可以在 [TiDB Cloud - 用户名前缀](/tidb-cloud/select-cluster-tier.md#user-name-prefix) 中获得更多信息。
```

File: TOC.md [hunk 5]
```diff
@@ -21,7 +21,7 @@
 - 应用开发
   - [概览](/develop/dev-guide-overview.md)
   - 快速开始
-    - [使用 TiDB Cloud (DevTier) 构建 TiDB 集群](/develop/dev-guide-build-cluster-in-cloud.md)
+    - [使用 TiDB Cloud (Developer Tier) 构建 TiDB 集群](/develop/dev-guide-build-cluster-in-cloud.md)
```
Comment (qiancai) [inline TOC.md:24]: 这个 PR 删除了三张图片，不确定是否会影响停止了构建的 dev 文档里三张图片的显示，可以请 @shczhen 确认下吗？谢谢

Comment (Icemap) [reply TOC.md:24]: 会有问题，我恢复一下删除的文件。

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 6]
```diff
@@ -16,17 +16,28 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
 
 1. 如果你还未拥有 TiDB Cloud 帐号，请先在此[注册](https://tidbcloud.com/free-trial)。
 2. 使用你的 TiDB Cloud 帐号[登录](https://tidbcloud.com/)。
-3. 在[方案](https://tidbcloud.com/console/plans)内选择一年内免费的 Developer Tier 方案，或直接点击[创建 Dev Tier 集群](https://tidbcloud.com/console/create-cluster?tier=dev)，进入 **Create a Cluster (Dev Tier)** 页面。
-4. 请在 **Create a Cluster (Dev Tier)** 页面填写集群名称/密码/云服务商（暂时仅可选择 AWS）/ 可用区（建议就近选择）后，点击 **Create** 按钮创建集群。
-5. 稍作等待，在 5~15 分钟后，将创建完毕，可在 [Active Clusters](https://tidbcloud.com/console/clusters) 查看创建进度。
+3. 在[方案](https://tidbcloud.com/console/plans)内选择一年内免费的 **Developer Tier** 方案。
+4. 在 **Create Cluster** 页面设置集群名称、云服务商（Developer Tier 默认为 AWS）、可用区（建议就近选择）后，点击 **Create** 按钮创建 Developer Tier 免费集群。
+
```
Comment (Icemap) [inline develop/dev-guide-build-cluster-in-cloud.md:21]:
```suggestion
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 7]
```diff
@@ -16,17 +16,20 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
...
-9. 复制弹出框 **Step 2: Connect with a SQL client** 中的连接字符串，供后续步骤使用。
-
-![SQL string](/media/develop/IMG_20220331-232800929.png)
+3. 在[方案](https://tidbcloud.com/console/plans)内选择一年内免费的 **Developer Tier** 方案，或在 [Active Clusters](https://tidbcloud.com/console/clusters) 页面中点击 **Create Cluster** 按钮。
+4. 在 **Create Cluster** 页面设置集群名称、云服务商（Developer Tier 默认为 AWS）、可用区（建议就近选择）后，点击 **Create** 按钮创建 Developer Tier 免费集群。
+5. 在 **Security Settings** 对话框中，设置密码，并添加允许连接你的集群的 IP 地址，完成后点击 **Apply**。
+
+    你的 TiDB Cloud 集群将于 5~15 分钟后创建完毕。
+
+6. 创建完毕后，直接点击右上角的 **Connect** 按钮。或点击集群名称，打开集群的详情页，再点击右上角的 **Connect** 按钮。这将显示一个连接对话框。
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:25]:
```suggestion
6. 创建完毕后，点击右上角的 **Connect** 按钮。或点击集群名称，打开集群的详情页，再点击右上角的 **Connect** 按钮。这将显示一个连接对话框。
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 8]
```diff
@@ -160,7 +153,7 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
     {{< copyable "shell-regular" >}}
 
     ```shell
-    mysql --connect-timeout 15 -u root -h xxx.tidbcloud.com -P 4000 -p
+    mysql --connect-timeout 15 -u '4JC1i9KroBMFRwW.root' -h xxx.tidbcloud.com -P 4000 -D test -p
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:156]:
```suggestion
    mysql --connect-timeout 15 -u '9ATyn6DhCXoo6U1.root' -h xxx.tidbcloud.com -P 4000 -D test -p
```

Comment (qiancai) [reply develop/dev-guide-build-cluster-in-cloud.md:156]: 要不改成前面 note 里提到的这个前缀

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 9]
```diff
@@ -171,14 +164,10 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
     mysqlDataSource.setServerName("xxx.tidbcloud.com");
     mysqlDataSource.setPortNumber(4000);
     mysqlDataSource.setDatabaseName("test");
-    mysqlDataSource.setUser("root");
+    mysqlDataSource.setUser("4JC1i9KroBMFRwW.root");
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:167]:
```suggestion
    mysqlDataSource.setUser("9ATyn6DhCXoo6U1.root");
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 10]
```diff
@@ -16,17 +16,20 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
...
+    你的 TiDB Cloud 集群将于 5~15 分钟后创建完毕。
+
+6. 创建完毕后，点击右上角的 **Connect** 按钮。或点击集群名称，打开集群的详情页，再点击右上角的 **Connect** 按钮。这将显示一个连接对话框。
+7. 复制连接对话框 **Step 2: Connect with a SQL client** 中的连接字符串，供后续步骤使用。
+
+    ![SQL string](/media/develop/tidb-cloud-connect.png)
+
+    > **Note:**
+    >
+    > - 需要特别说明的是，在你使用 [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier) 集群时，你需要给你设置的用户名加上前缀（如上图中的 `9ATyn6DhCXoo6U1`），若使用命令行连接，还需使用单引号包裹用户名。你可以在 [TiDB Cloud - 用户名前缀](/tidb-cloud/select-cluster-tier.md#user-name-prefix) 中获得更多信息。
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:32]:
```suggestion
    > 需要特别说明的是，在你使用 [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier) 集群时，你需要给你设置的用户名加上前缀（如上图中的 `9ATyn6DhCXoo6U1`），若使用命令行连接，还需使用单引号包裹用户名。你可以在 [TiDB Cloud - 用户名前缀](/tidb-cloud/select-cluster-tier.md#user-name-prefix) 中获得更多信息。
```

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 11]
```diff
@@ -16,17 +16,20 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
...
+    你的 TiDB Cloud 集群将于 5~15 分钟后创建完毕。
+
+6. 创建完毕后，点击右上角的 **Connect** 按钮。或点击集群名称，打开集群的详情页，再点击右上角的 **Connect** 按钮。这将显示一个连接对话框。
+7. 复制连接对话框 **Step 2: Connect with a SQL client** 中的连接字符串，供后续步骤使用。
+
+    ![SQL string](/media/develop/tidb-cloud-connect.png)
+
+    > **Note:**
+    >
+    > 需要特别说明的是，在你使用 [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier) 集群时，你需要给你设置的用户名加上前缀（如上图中的 `9ATyn6DhCXoo6U1`），若使用命令行连接，还需使用单引号包裹用户名。你可以在 [TiDB Cloud - 用户名前缀](/tidb-cloud/select-cluster-tier.md#user-name-prefix) 中获得更多信息。
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:32]:
```suggestion
    > 需要特别说明的是，在你使用 [Developer Tier clusters](https://docs.pingcap.com/tidbcloud/select-cluster-tier#developer-tier) 集群时，你需要给你设置的用户名加上前缀（如上图中的 `9ATyn6DhCXoo6U1`），若使用命令行连接，还需使用单引号包裹用户名。你可以在 [TiDB Cloud - 用户名前缀](https://docs.pingcap.com/tidbcloud/select-cluster-tier#user-name-prefix) 中获得更多信息。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**